pwa/src/templates/templateParts/submitComponent/SubmitComponentTemplate.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import * as styles from "./SubmitComponentTemplate.module.css";
+import { ArrowRightIcon } from "@gemeente-denhaag/icons";
+import { Button } from "@gemeente-denhaag/components-react";
+import clsx from "clsx";
+import { navigate } from "gatsby";
+
+interface SubmitComponentTemplateProps {
+  layoutClassName?: string;
+}
+
+export const SubmitComponentTemplate: React.FC<SubmitComponentTemplateProps> = ({ layoutClassName }) => {
+  return (
+    <div className={clsx(styles.container, layoutClassName && layoutClassName)}>
+      <div className={styles.header}>
+        <div className={styles.title}>Aan de slag met Open Catalogi</div>
+
+        <div>
+          Wilt u uw component op Open Catalogi aanbieden zodat andere uw component kunnen (her)gebruiken of bij dragen
+          aan de doorontwikkeling van uw component?
+        </div>
```

Comment:
I'd probably make this a `span` instead of a `div`, but doesn't matter too much.

I would however add some line-height to this element, because the text is very compressed.

I think `--web-app-line-height-sm` would be good 👍🏼